Gems/PhysX/Code/Source/EditorRigidBodyComponent.h

```diff
@@ -76,14 +76,16 @@ namespace PhysX
 
         static void GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent)
         {
-            dependent.push_back(AZ_CRC_CE("PhysicsColliderService"));
             dependent.push_back(AZ_CRC_CE("NonUniformScaleService"));
         }
 
         // AZ::Component
         void Activate() override;
         void Deactivate() override;
 
+        // AZ::EntityBus overrides...
```

Comment:
nitpick - I think there's usually a space between overrides and ...

Reply:
I think I was not adding space there normally... but I can add it.